src/components/Input/Input.test.tsx

```diff
@@ -215,36 +215,6 @@ describe("Forwarding refs", () => {
   });
 });
 
-describe("Radio buttons", () => {
```

Comment:
The Input component will eventually be deprecated, so I am not sure these changes are necessary.

Reply:
@bigfishdesign13 You're right. I did this so that I could run the test suite locally and not have it error.  Since the changes were already done a while back, I would recommend keeping as-is for now.  Thoughts?